base/proc.dtx

```diff
@@ -340,7 +340,17 @@
 \let\@evenhead\@empty
 \def\@oddfoot{\normalfont\rightmark \hfil
               \pagename{} \thepage}
-\def\@evenfoot{\@oddfoot}
+%    \end{macrocode}
+%    If the \texttt{twoside} option is passed to the classe we provide
+%    a different footer that shows the \cs{leftmark} if that is set.
+% \changes{v1.3m}{2021/12/09}{Support twoside (gh/704)}
+%    \begin{macrocode}
+\if@twoside
+  \def\@evenfoot{\normalfont\pagename{} \thepage
```

Comment:
the OR has done \reset@font so \normalfont shouldn't really be needed (but proc has apparently always used \normalfont in the foot so perhaps do same here. If its removed same comment applies to the existing setting of \@oddfoot  at line 341 above)

Reply:
might as well remove it